gnat2goto/driver/goto_utils.adb

```diff
@@ -532,11 +619,108 @@ package body GOTO_Utils is
       if Kind (Follow_Symbol_Type (Get_Type (Expr),
                A_Symbol_Table)) = I_C_Enum_Type
       then
-         return Typecast_If_Necessary (Expr,
-                                       Make_Signedbv_Type (32),
-                                       A_Symbol_Table);
+         return Typecast_If_Necessary (Expr, C_Int_T, A_Symbol_Table);
       else
          return Expr;
       end if;
    end Cast_Enum;
+
+   function Make_Assert_Call (Assertion : Irep;
+                              Description : Irep; Source_Loc : Source_Ptr)
+                              return Irep is
+      SE_Call_Expr : constant Irep :=
+        New_Irep (I_Code_Function_Call);
+      Sym_Assert   : constant Irep := New_Irep (I_Symbol_Expr);
+      Assert_Args  : constant Irep := New_Irep (I_Argument_List);
+   begin
+      Set_Identifier (Sym_Assert, "__CPROVER_assert");
```

Comment:
Could you replace those with regular constructor calls?

Reply:
Done: there should be no `new_irep` in the added code.